pandas/tests/series/test_arithmetic.py

```diff
@@ -79,6 +79,22 @@ def test_ser_cmp_result_names(self, names, op):
 
 
 class TestTimestampSeriesComparison(object):
+    def test_dt64ser_cmp_date_invalid(self):
+        # GH#19800 datetime.date comparison raises to
```

Comment:
I think we had a reference to the python docs about this, can you add it here. IOW a future reader may not know this.

Reply:
I don't see this mentioned [here](https://docs.python.org/3/library/datetime.html).  Is there another doc you have in mind?

Reply:
I don't recall, but this is a 'feature' in Python. Need a reference to it. For the next reader.